### PR TITLE
Tsc 485 event search minor changes

### PR DESCRIPTION
**Pull request by jay-jhk-lee**
Changed label of "status" column to "column on?"
changed label of "qualifier" to "type"
changed time display to a table for line up

image

### CODE REVIEW

**Comment by paologum** [comment]
it probably would make more sense to align text to the left
## `text-align: left`

Screenshot 2024-08-14 at 4 12 37 PM

## `text-align: right`
Screenshot 2024-08-14 at 4 12 51 PM

**Comment by paologum** [comment]
probably atp we might just want to have two cells for `Top` and `Base` and potentially put it on the left or right but we can do that/reconvene on that later

**Comment by jay-jhk-lee** [comment]
changed alignment of times
image

File: app/src/settings_tabs/Search.css [hunk 1]
```diff
@@ -13,35 +13,25 @@
 }
 
 .search-time-display-container {
-  min-width: 10vw;
-  padding-left: 20px;
+  min-width: 200px;
+  min-height: 70px;
+  resize: both;
```
Comment (paologum) [inline app/src/settings_tabs/Search.css:18]: you should add a `max-height` and `max-width` so that they can't expand it to the whole page

**Review by paologum — CHANGES_REQUESTED** [review]

**Review by paologum — APPROVED** [review]

**Pull request merged**